doc/man/8/monmaptool.rst

```diff
@@ -9,8 +9,16 @@
 Synopsis
 ========
 
-| **monmaptool** *mapfilename* [ --clobber ] [ --print ] [ --create ]
-  [ --add *ip*:*port* *...* ] [ --addv [*protocol*:*ip*:*port*[,*...*] ] *...* ] [ --rm *ip*:*port* *...* ]
+| **monmaptool** [--print] [--create [--clobber] [--fsid *fsid*]]
+  [--enable-all-features]
+  [--generate] [--set-initial-members]
+  [--add *name* *ip:port*] [--rm *name*]
+  [--addv *name* [v2:*ip:port*,v1:*ip:port*]]
+  [--feature-list [plain|parseable]]
+  [--feature-set *value* [--optional|--persistent]]
+  [--feature-unset *value* [--optional|--persistent]]
+  [--set-min-mon-release *release*]
+  *mapfilename*
```

Comment:
how about something like
```
| **monmaptool** <action> [options] *mapfilename*
```
?

Reply:
@tchaikov Do not you think a more explicit synopsis is better ? It also matches what we have on other pages (ceph-osd(8), for instance)

Reply:
Alexandre, i was rethinking about what a **synopsis** should really look like. by looking at the pile of usage, one cannot find out the first argument is actually "action" and what follows are the options for the action very quickly. and the explicit version of synopsis just repeats the different options.

regarding to consistency with other man pages, i concur with you. it's indeed an issue. if you feel strong about this, i have no objections (as i've approved this pull request). i just wanted to understand what you think about a concise synopsis.

Reply:
@tchaikov Thanks you for the feedback

I've paid more attention to manpages from other projects as well as asked people what they like, and it seems you are right

I have updated the commit with the changes, and updated other outdated parts as well

Reply:
merci bcp!